src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -3233,4 +3239,30 @@ namespace cryptonote
     };
     typedef epee::misc_utils::struct_init<response_t> response;
   };
+
+
+  LOKI_RPC_DOC_INTROSPECT
+  struct COMMAND_RPC_REPORT_PEER_SS_STATUS
+  {
+    struct request
+    {
+      std::string type; // test type (currently used: ["reachability"])
```

Comment:
This field isn't being used, you can probably remove it and re-add it when additional tests are available since only internal code is using this we're not going to break any external code.

Reply:
I don't want to deal with incompatibility issues in the future (even "internal"), so I changed the code to actually use the *type* value instead.